### DISCUSSION TITLE
index.css breaks if @layer code is formatted

### DISCUSSION THREAD

**Discussion by robertgr991** [chakra-ui/panda#925] [Q&A] [answered]
Hello, `index.css` seems to break if the code inside is not exactly `@layer reset, base, tokens, recipes, utilities;` without any modification.

Prettier seems to format this file to:
```css
@layer reset,
base,
tokens,
recipes,
utilities;
```
I followed the Solid.js starting guide. Maybe there should be an aside in docs that says that this file contents shouldn't be modified at all or change the generator to handle content normalization.

**Comment by segunadebayo** [comment]
The layer declaration is a valid css syntax, and prettier should not format those. I don't experience this in my local prettier.

Please consider updating the prettier settings in your project.

**Comment by robertgr991** [comment]
I did update the prettier config and now it handles it fine, but I still think it's worth at least mentioning in the docs that the code snippet should be exactly the same, it's easy to get it wrong.

Edit: The way prettier formatted it initially is still valid CSS syntax, that's why I think it should be specified.

**Reply by astahmer** [reply, marked as answer]
we had a strict check on it, it should be fixed
https://github.com/chakra-ui/panda/pull/1009
